index.rst

```diff
@@ -1,70 +1,87 @@
-Welcome to Aiven Developer Portal
-=================================
+Aiven Developer
+===============
 
-Aiven offers managed services for the best open source data technologies, on a cloud of your choice.
+What can we help you with today?
 
-Clouds
-------
+.. raw:: html
 
-We offer multiple cloud options because we believe that everyone should have access to great data platforms to host their applications. Our customers tell us they love the multi-cloud approach because they know that they aren't locked in to one particular cloud platform for all their data needs.
+    <form class="sidebar-search-container" method="get" action="search.html">
+      <input class="sidebar-search" placeholder=Search name="q">
+      <input type="hidden" name="check_keywords" value="yes">
+      <input type="hidden" name="area" value="default">
+    </form>
 
-You can choose from Amazon Web Services (AWS), Google Cloud Platform (GCP), Microsoft Azure Cloud, DigitalOcean Cloud and UpCloud, and then pick the region where your service should be hosted.
 
 
-Pricing
--------
+(Looking for the previous documentation? Visit `Aiven Help <https://help.aiven.io>`_ or use the search above.)
 
-You are billed on a monthly basis, and you only pay for the time that the service is running. Data transfer costs are included. 
-You can check more on our `Pricing page <https://aiven.io/pricing>`_.
+----------------
 
-Products
---------
+Explore
+-------
 
-Relational databases that you know and love ❤️
+Learn about the Aiven platform
 
 .. panels::
+    :card: shadow
+
+    |icon-postgres| **PostgreSQL** Powerful relational database platform. We have the latest versions, and all the extensions you need.
```

Comment:
"all extensions you need" <- isn't this a bit over-promising?

Reply:
I liked it! But you are probably correct, I'll tone it down.